pyhatchbabyrest/pyhatchbabyrestasync.py

```diff
@@ -141,6 +149,12 @@ async def set_brightness(self, brightness):
             self.color[0], self.color[1], self.color[2], brightness
         )
         return await self._send_command(command)
+    
+    async def set_time(self, time: struct_time = None):
```

Comment:
The type hint for `time` should be `Optional[struct_time]`.

If we add newer annotations, it would be `struct_time|None`.